i18n/zh/docusaurus-theme-classic/navbar.json

```diff
@@ -46,5 +46,13 @@
   "item.label.v1.13": {
     "message": "v1.13",
     "description": "Navbar item with label v1.13"
+  },
+  "item.label.Next": {
+    "message": "Next",
+    "description": "Navbar item with label Next"
+  },
+  "item.label.v1.15": {
+    "message": "v1.15",
+    "description": "Navbar item with label v1.15"
```

Comment:
Another question, what's meaning of labels above? And what's the effect without them?

Reply:
> Another question, what's meaning of labels above? And what's the effect without them?

Auto generated Chinese version of the label name using `yarn write-translations`. Actually, when we do any changes on the pages, we need to using `yarn write-translations --locale zh` to generate the corresponding Chinese version and translate it. 
For example, if I change the "message" content to "1.15 版本" now, it would be displayed in Chinese as follows. 
![image](https://github.com/kubeedge/website/assets/48788150/f271cb14-d543-4542-8e98-a5f2d20f91e5)

Reply:
Ok, sounds good, if it's auto generated, how about adding a check in CI?

Reply:
> Ok, sounds good, if it's auto generated, how about adding a check in CI?

Maybe it's worth to add this CI to check the Chinese version. I will create a issue to track.

Reply:
Okey. Let's open an issue to track it.

Reply:
> Okey. Let's open an issue to track it.

#489